### ISSUE TITLE
[Feature Request] Allow setting cryfs block size

### ISSUE THREAD

**Issue by automorphism88** [hardcore-sushi/DroidFS#278]
It appears that CryFS volumes created with DroidFS always use the default block size (currently 16k). But cryfs allows users to specify the block size with the --blocksize command line argument. Depending on the nature of the files stored and the underlying filesystem and device, users might have good reason to want to use a different block size. For example, for an exFAT filesystem, the usual block size is 128k. A CryFS volume stored there which uses the default block size of 16k will be wasting 7/8ths of the space in each filesystem block, since it will be creating files of size 16k that take up an entire 128k block on disk. In such a case, it would be far more space-efficient to use --blocksize 131072 when creating the CryFS filesystem. It seems that this is a primary use case for DroidFS, given that on an unrooted phone, you can't mount external storage formatted with ext4 or f2fs.

**Comment by hardcore-sushi** [comment]
Yes. It's also tracked there: [#14](https://forge.chapril.org/hardcoresushi/DroidFS/issues/14).
